backend/lcfs/web/api/compliance_report/update_service.py

```diff
@@ -219,6 +219,7 @@ async def handle_assessed_status(self, report: ComplianceReport):
             raise HTTPException(status_code=403, detail="Forbidden.")
         # Update the transaction to assessed
         report.transaction.transaction_action = TransactionActionEnum.Adjustment
+        report.transaction.update_user = str(user.user_profile_id)
```

Comment:
update_user and create_user should be handled correctly but the auditing dependencies. It is also set to the users username, not their ID.

Reply:
modified to set the username instead of user id